Common/AR/source/ArcGISArViewInterface.cpp

```diff
@@ -392,10 +413,9 @@ void ArcGISArViewInterface::updateTrackingSources()
 {
   if (m_tryUsingArKit)
   {
-    // disable LocationDataSource if necessary.
-    // Don't delete these objects, they can be the user's objects.
-    disconnect(m_locationChangedConnection);
-    disconnect(m_headingChangedConnection);
+    // Stop locationDataSource is exist.
```

Comment:
```suggestion
    // Stop locationDataSource if it exists
```

Reply:
Add a closing period to James' suggestion:

 // Stop locationDataSource if it exists.